packages/client/hmi-client/src/components/workflow/ops/model-config/tera-model-config-drilldown.vue

```diff
@@ -633,6 +619,20 @@ const resetConfiguration = () => {
 	});
 };
 
+watch(
+	() => props.node.state.documentModelConfigTaskId,
+	() => {
+		documentModelConfigTaskId.value = props.node.state.documentModelConfigTaskId ?? '';
+	}
+);
+
+watch(
+	() => props.node.state.datasetModelConfigTaskId,
+	() => {
+		datasetModelConfigTaskId.value = props.node.state.datasetModelConfigTaskId ?? '';
+	}
+);
```

Comment:
I think e can clean up things a bit here with a string trick, instead of having so many watchers and trying to manage so many variables....we cao do something like

```
const isLoading = ref(false)
...

watch(
	() => props.node.state.datasetModelConfigTaskId + ": " + props.node.state.datasetModelConfigTaskId
	(watchStr) => {
    if (watchStr !== ':') isLoading.value = true;
    else isLoading.value = false;
	}
);
```

Then we can remove one watcher, and remove the extra refs documentModelConfigTaskId and datasetModelConfigTaskId